lib/cmake/scipp-conan.cmake

```diff
@@ -28,55 +28,52 @@ include(${CMAKE_CURRENT_BINARY_DIR}/conan.cmake)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_BINARY_DIR})
 list(APPEND CMAKE_PREFIX_PATH ${CMAKE_CURRENT_BINARY_DIR})
 
-option(CONAN_TBB "Use TBB from conan" OFF)
-if(SKBUILD OR CONAN_TBB)
-  # Conda builds install tbb via conda, not conan
-  set(CONAN_ONETBB onetbb/2021.12.0)
+if(SKBUILD)
   # The deploy generator is used to install dependencies into a know location.
   # The RUNTIME_DEPENDENCIES DIRECTORIES install option is then used to find the
   # dependencies. This is actually required only for Windows, since on Linux and
   # OSX cmake finds the files anyway, based on the target's rpath.
-  set(CONAN_DEPLOY deploy)
   set(CONAN_RUNTIME_DEPENDENCIES "tbb")
-endif()
-
-if(BENCHMARK)
-  set(CONAN_BENCHMARK benchmark/1.6.1)
+  conan_cmake_configure(
+    REQUIRES
+    benchmark/1.6.1
+    boost/1.86.0
+    eigen/3.4.0
+    gtest/1.11.0
+    LLNL-Units/0.9.1
+    pybind11/2.13.5
+    onetbb/2021.12.0
+    OPTIONS
+    benchmark:shared=False
+    boost:header_only=True
+    gtest:shared=False
+    LLNL-Units:shared=False
+    LLNL-Units:fPIC=True
+    LLNL-Units:base_type=uint64_t
+    LLNL-Units:namespace=llnl::units
+    GENERATORS
+    cmake_find_package_multi
+    deploy
+  )
 else()
-  set(CONAN_BENCHMARK "")
+  conan_cmake_configure(
+    REQUIRES
+    LLNL-Units/0.9.1
+    OPTIONS
+    LLNL-Units:shared=False
+    LLNL-Units:fPIC=True
+    LLNL-Units:base_type=uint64_t
+    LLNL-Units:namespace=llnl::units
+    GENERATORS
+    cmake_find_package_multi
+  )
```

Comment:
I gather you have not (yet?) switched to using the conan package you made for `units`?

Reply:
no, it won't actually work for us as the maintainers of conancenter didn't want to deviate from upstream so the one available from conan has min cpp version at 14, and we would need to have a custom recipe anyway to keep the min cpp version at 17 to use our namespace.